src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix filter applied on constant names
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
```

Comment:
well.. it's a conceptual issue off course :) however if we can improve it in an abstract manner.. i think we should.

More or less related to what @stof mentioned in https://github.com/symfony/symfony/pull/22686#discussion_r115770933